src/main/java/com/commercetools/project/sync/model/request/ResourceIdsGraphQlRequest.java

```diff
@@ -0,0 +1,78 @@
+package com.commercetools.project.sync.model.request;
```

Comment:
I guess request is not necessary as we don't have many models on this project, because 'model' is an empty package already, would it better to change package to `package com.commercetools.project.sync.model` ?

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/234/commits/04238773a78e8c27521603eb4be93cf53bcee340

Reply:
> would it better to change package to package com.commercetools.project.sync.model ?
now it is com.commercetools.project.sync.request :D

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/234/commits/8970492b2a2794bb162b373fe125edfd13a82ca7